spark/src/test/scala/org/apache/mahout/sparkbindings/test/DistributedSparkSuite.scala

```diff
@@ -58,6 +58,11 @@ trait DistributedSparkSuite extends DistributedMahoutSuite with LoggerConfigurat
   }
 
 
+  override protected def afterAll(configMap: ConfigMap): Unit = {
```

Comment:
oh. this is why we kept growing unused contexts in tests? 

if so, good catch. thank you.